eng/common/TestResources/New-TestResources.ps1

```diff
@@ -292,7 +301,6 @@ try {
         }
     }
 
-    # This needs to happen after we set the TenantId but before we use the ResourceGroupName
```

Comment:
Looks like you lost this change from the move as well.

Reply:
thanks